frontend/src/components/KeyMeasureDiagnostic/ExpeModals/ExpeVegetarian.vue

```diff
@@ -0,0 +1,788 @@
+                  <v-card-text>
+                    <!-- Vegetarian menu rate -->
+                    <label
+                      class="body-2 grey--text text--darken-3 font-weight-medium"
+                      :for="`vegetarian-menu-percentage-${item.value}`"
+                    >
+                      Pourcentage de menus végétariens servis par rapport aux autres menus
+                    </label>
+                    <v-text-field
+                      validate-on-blur
+                      hide-details="auto"
+                      :rules="[validators.nonNegativeOrEmpty]"
+                      solo
+                      dense
+                      v-model.number="expe[`vegetarianMenuPercentage${item.value}`]"
+                      append-icon="mdi-percent"
+                      style="max-width: 150px"
+                      class="mt-2 mb-4 body-2"
+                      :id="`vegetarian-menu-percentage-${item.value}`"
+                    ></v-text-field>
+
+                    <!-- Menu composition -->
+                    <label class="body-2 grey--text text--darken-3 font-weight-medium">
+                      Dans votre plan alimentaire de 20 repas successifs, les options végétariennes servies dans votre
+                      établissement sont des plats à base de :
+                    </label>
+                    <div class="mt-4 mb-8" style="border-bottom: solid 1px #EEE;">
+                      <div
+                        class="d-flex py-2"
+                        style="border-top: solid 1px #EEE;"
+                        v-for="category in categories"
+                        :key="category.htmlId"
+                      >
+                        <label
+                          class="body-2 grey--text text--darken-3 pr-2"
+                          :style="$vuetify.breakpoint.mdAndUp ? 'width: 35%;' : 'width: 70%;'"
+                          :for="`${category.htmlId}-${item.value}`"
+                        >
+                          {{ category.label }}
+                        </label>
+                        <div class="d-flex">
+                          <v-text-field
+                            validate-on-blur
+                            hide-details="auto"
+                            solo
+                            dense
+                            v-model.number="expe[`${category.fieldName}${item.value}`]"
+                            :rules="[
+                              validators.nonNegativeOrEmpty,
+                              validators.lteSumValue(
+                                self[`compositionValidationFields${item.value}`],
+                                20,
+                                compositionSumErrorMessage
+                              ),
+                            ]"
+                            :id="`${category.htmlId}-${item.value}`"
+                          ></v-text-field>
+                          <div class="my-auto ml-2">/20</div>
```

Comment:
Maybe you can use a suffix on the text field for this https://vuetifyjs.com/en/api/v-text-field/#props-suffix

Reply:
I should've known it was somewhere in the API! :smile:  Fix here : https://github.com/betagouv/ma-cantine/pull/1237/commits/f5a4fd93eeb2d52678a6456a47b13533b9b75da6